extensionsdk/Microsoft.Windows.DevHome.SDK/ComputeSystemOperationData.cpp

```diff
@@ -0,0 +1,20 @@
+#include "pch.h"
```

Comment:
All extensionsdk changes need to be made separately.  Are these already checked in and github diff is just not picking it up correctly?

Reply:
huh not sure why this made it in. will remove. It is from an older PR, a month ago. There shouldn't be any SDK changes in this PR.

Reply:
removed